server/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
@Lemore I'm not sure way you commit this file, there is no need for one. Pls remove it.